src/app/client/package.json

```diff
@@ -133,7 +133,8 @@
     "videojs-contrib-quality-levels": "^2.1.0",
     "videojs-http-source-selector": "^1.1.6",
     "zone.js": "~0.10.2",
-    "ngtek-event-library": "^1.0.174"
+    "ngtek-event-library": "^1.0.174",
+    "ngx-pagination": "^5.1.1"
```

Comment:
Pending Items (Need to work on it)
- Use Sunbird Pagination
- Use Joyful and Classic theme classes